src/CukeCommands.cpp

```diff
@@ -1,6 +1,10 @@
 #include "cucumber-cpp/internal/CukeCommands.hpp"
 
-#include <sstream>
+#if __cplusplus <= 199711L
+    #include <vector>
+#else
+    #include <array>
+#endif
```

Comment:
I'm not in favour of duplicating the test that needs to be done to test both implementations. For now I'd stick to not using C++11. In the long run I'd rather do dynamic loading of step definitions so that they can be compiled separately from the core and convert it to C++11.